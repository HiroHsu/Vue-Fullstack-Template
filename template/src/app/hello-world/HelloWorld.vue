<template>
  <v-container fluid>
    <v-layout>
      <div class="title">Hello, World!</div>
    </v-layout>
  </v-container>
</template>

<script>
export default {
  metaInfo: {
    title: 'Hello World | Vue by Example',
  },
};
</script>

<style scoped>
</style>
