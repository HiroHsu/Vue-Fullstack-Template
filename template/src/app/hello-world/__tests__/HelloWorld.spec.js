import { shallow } from '@vue/test-utils';

import HelloWorld from '../HelloWorld';

describe('Hello World', () => {
  let [wrapper] = [];

  beforeEach(() => {
    wrapper = shallow(HelloWorld);
  });

  it('should render initial component', () => {
    expect(wrapper.html()).toMatchSnapshot();
  });
});
