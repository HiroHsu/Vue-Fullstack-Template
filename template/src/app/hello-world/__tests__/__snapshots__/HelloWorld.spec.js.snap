// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Hello World should render initial component 1`] = `
<v-container fluid="">
  <v-layout>
    <div class="title">Hello, World!</div>
  </v-layout>
</v-container>
`;
